packages/Molecules/Card/stories.js

```diff
@@ -34,6 +35,13 @@ storiesOf('Molecules/Card', module)
       return demoData.cardNavigationFeatured()
     }
   })))
+  .add('Card Masonry', withReadme(readme, () => ({
+    components: { RplCardMasonry },
+    template: `<rpl-card-masonry :title="title" :summary="summary" :link="link" :image="image" />`,
+    data() {
```

Comment:
Hey @komangariana can you please add a space before function parentheses? `data ()`. It throw a Eslint waring at the moment.